### PR TITLE
[Compute AG: CWP-29369]: Add registry scanning page for GitLab

### PR DESCRIPTION
**Pull request by divyabhushan-okta**
## Reference

[CWP-29369](https://redlock.atlassian.net/browse/CWP-29369)

### CODE REVIEW

File: compute/admin_guide/vulnerability_management/registry_scanning/scan-gitlab.adoc [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+.. Enter *Tag* numbers to scan, leave blank, or enter a wildcard (*) to scan all the tags.
+
+.. Optionally, enter *Tags to exclude*, to avoid scanning images with specified tags.
+
+.. Provide at least one of the fields:
+... *User ID*: GitLab user account
+... *Group ID*: Enter a single group ID, or a list of group IDs
+... *Project ID*: Enter a GitLab Project ID, or a list of project IDs
++
+NOTE: To scan all the registries, enter only the *User ID*. When Project ID or Group ID are provided, the registry scan will be triggered accordingly.
```
Comment (divyabhushan-okta) [inline compute/admin_guide/vulnerability_management/registry_scanning/scan-gitlab.adoc]: Also mention - when all the 3 are given, the User ID takes precedence.

Comment (divyabhushan-okta) [reply compute/admin_guide/vulnerability_management/registry_scanning/scan-gitlab.adoc]: Discard the above. Corrected sentence: "When we have all together- we will query the project and the group ids (will not use the user id)"

File: compute/admin_guide/vulnerability_management/registry_scanning/scan-gitlab.adoc [hunk 2]
```diff
@@ -0,0 +1,72 @@
+== Scan Images in GitLab Container Registry
+
+Configure Prisma Cloud to scan your GitLab Container Registry on GitLab EE without using administrator credentials.
```
Comment (divyabhushan-okta) [inline compute/admin_guide/vulnerability_management/registry_scanning/scan-gitlab.adoc]: To be confirmed by Engineering, if we support scanning on just GitLab EE or on CE as well!

File: compute/admin_guide/vulnerability_management/registry_scanning/scan-gitlab.adoc [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+... *Project ID*: Enter a GitLab Project ID, or a list of project IDs. The project ID is used to locate all the registries located within a specific project.
++
+[NOTE]
+====
+- To trigger a full scan, including all repositories of a user across every group and project, enter just the user ID.
+- When you enter all the IDs, Prisma will query the project IDs and the group IDs, the user ID won't be queried.
+- When you enter both user ID and the group ID, Prisma will query the registries based on the group ID.
+- With both user ID and project ID, Prisma will query based on the project ID.
+- When both group ID and project ID are provided, Prisma queries both the IDs.
+====
```
Comment (divyabhushan-okta) [inline compute/admin_guide/vulnerability_management/registry_scanning/scan-gitlab.adoc:52]: @Pubs-MV, The main use-case is to scan the entire repository. Should I include the other combinations or not! WDYT

File: compute/admin_guide/vulnerability_management/registry_scanning/scan-gitlab.adoc (hunk 2 above)
Comment (divyabhushan-okta) [reply compute/admin_guide/vulnerability_management/registry_scanning/scan-gitlab.adoc]: GCR scan is supported on both EE and CE.

File: compute/admin_guide/vulnerability_management/registry_scanning/scan-gitlab.adoc [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+.. In *Repository types*, select the repository types that Prisma Cloud should scan.
+
+.. Enter *Tag* numbers to scan, leave blank, or enter a wildcard (*) to scan all the tags.
+
+.. Optionally, enter *Tags to exclude*, to avoid scanning images with specified tags.
+
+.. Provide at least one of the fields:
+... *User ID*: GitLab user account. The user ID is used to get all registries associated with the user.
+... *Group ID*: Enter a single group ID, or a list of group IDs. The group ID is used to locate all the registries within a specific group.
+... *Project ID*: Enter a GitLab Project ID, or a list of project IDs. The project ID is used to locate all the registries located within a specific project.
```
Comment (ochitayat) [inline compute/admin_guide/vulnerability_management/registry_scanning/scan-gitlab.adoc:42]: Please add the Repository field as well.
The user need to provide at least one of the following fields: Repository, User ID, Group ID or Project ID.

File: compute/admin_guide/vulnerability_management/registry_scanning/scan-gitlab.adoc (hunk 2 above)
Comment (tallieber) [reply compute/admin_guide/vulnerability_management/registry_scanning/scan-gitlab.adoc]: so we need to change the sentence here, right?

File: compute/admin_guide/vulnerability_management/registry_scanning/scan-gitlab.adoc (hunk 4 above)
Comment (tallieber) [reply compute/admin_guide/vulnerability_management/registry_scanning/scan-gitlab.adoc:42]: @divyabhushan  This change will be on Newton. for now, one of the 3 fields are required: project, user or group id. 
cc @ochitayat

Comment (divyabhushan-okta) [reply compute/admin_guide/vulnerability_management/registry_scanning/scan-gitlab.adoc:42]: Thanks @tallieber noted.

File: compute/admin_guide/vulnerability_management/registry_scanning/scan-gitlab.adoc (hunk 2 above)
Comment (divyabhushan-okta) [reply compute/admin_guide/vulnerability_management/registry_scanning/scan-gitlab.adoc]: Thanks Tal, I updated it with just GitLab to indicate both EE and CE.

**Review by Pubs-MV — APPROVED** [review]

**Pull request merged**